From: Mel Gorman <mgorman@suse.de>
Date: Thu, 16 Jan 2014 10:38:19 +0000
Subject: [PATCH] mm: vmstat: Do not display stats for TLB flushes unless
 debugging

References: VM Performance
Patch-mainline: No (expected v3.14)

The patch "x86: mm: Account for TLB flushes only when debugging"
removed vmstat counters related to TLB flushes unless CONFIG_DEBUG_TLBFLUSH
was set. It removed the counters from the vm_event_item enum but not
the vmstat_text text.

Signed-off-by: Mel Gorman <mgorman@suse.de>
---
 mm/vmstat.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/mm/vmstat.c b/mm/vmstat.c
index 5a442a7..8e8acfa 100644
--- a/mm/vmstat.c
+++ b/mm/vmstat.c
@@ -851,12 +851,14 @@ const char * const vmstat_text[] = {
 	"thp_zero_page_alloc",
 	"thp_zero_page_alloc_failed",
 #endif
+#ifdef CONFIG_DEBUG_TLBFLUSH
 #ifdef CONFIG_SMP
 	"nr_tlb_remote_flush",
 	"nr_tlb_remote_flush_received",
-#endif
+#endif /* CONFIG_SMP */
 	"nr_tlb_local_flush_all",
 	"nr_tlb_local_flush_one",
+#endif /* CONFIG_DEBUG_TLBFLUSH */
 
 #endif /* CONFIG_VM_EVENTS_COUNTERS */
 };
